dashboard/app/models/resource.rb

```diff
@@ -26,18 +26,18 @@
 # @attr [Boolean] assessment - indicates whether this resource is an assessment
 # @attr [String] audience - who this resource is targeted toward (eg teacher, student, etc)
 # @attr [String] download_url - URL that can download the file
-# @attr [Boolean] printable_student_handout - indicates whether the file will be included in a PDF handout
-class Resource < ApplicationRecord
+# @attr [Boolean] include_in_pdf - indicates whether the file will be included in a PDF handout
+class Resource < ActiveRecord::Base
   include SerializedProperties
 
   has_and_belongs_to_many :lessons, join_table: :lessons_resources
   belongs_to :course_version
 
-  serialized_attrs %(
+  serialized_attrs %w(
     type
     assessment
     audience
     download_url
-    printable_student_handout
+    include_in_pdf
```

Comment:
I renamed this because I thought this was clearer, but let me know if this isn't correct.